broker/getters.go

```diff
@@ -7,17 +7,17 @@ import "github.com/TheThingsNetwork/ttn/core/types"
 
 func (m *DeviceActivationRequest) GetAppEUI() (eui types.AppEUI) {
 	if m != nil {
-		if m.AppEUI != nil {
-			return *m.AppEUI
+		if !m.AppEUI.IsEmpty() {
```

Comment:
No need for this `if`. We're returning the `Empty` eui anyway. Similar changes below.